### DISCUSSION TITLE
error upload file

### DISCUSSION THREAD

**Discussion by jossmanbr2022** [pocketbase/pocketbase#850] [Q&A] [answered]
I have a question hope you can help me
I need to upload to my db 3 pdf for the same user
I am using svelte:
pdfCFDI = new FormData();
on:change={(e)=>{
                     let files = e.target.files[0];
                       pdfCFDI.append('file', files);
                         }};

I repeat this in the 3 input type file at the end to upload my file I use something like this:
await client.records.create(collection, {
other data,
pdfCFDI
});
But this does not work for me because the pdfCFDI appears as empty

**Comment by ganigeorgiev** [comment, marked as answer]
@jossmanbr2022 It looks like you are mixing json and FormData. You have to use only FormData when uploading files:

```js
await client.records.create(collection, pdfCFDI);
```

_You can set your other regular fields in the FormData object similarly using append, eg. `pdfCFDI.append("title", "example")`_

**Reply by jossmanbr2022** [reply]
Checking my code if I was confusing the json with the formData thanks for your help
